pyEpiabm/pyEpiabm/tests/test_sweep/test_update_place_sweep.py

```diff
@@ -1,54 +1,95 @@
 import unittest
+from unittest import mock
 
-import pyEpiabm as pe
+from pyEpiabm.property.place_type import PlaceType
+from pyEpiabm.core.population import Population
+from pyEpiabm.core.parameters import Parameters
+from pyEpiabm.sweep.update_place_sweep import UpdatePlaceSweep
```

Comment:
```suggestion
from pyEpiabm.property import PlaceType
from pyEpiabm.core import Parameters, Population
from pyEpiabm.sweep import UpdatePlaceSweep
```
Shouldn't cause any import errors, but always best to import from subpackage rather than file!